src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -411,52 +318,37 @@ frmelmnts.lbl.name = Name
 frmelmnts.lbl.nameRequired = Name is required
 frmelmnts.lbl.next = Next
 frmelmnts.lbl.noContentToPlay = No content to play
-frmelmnts.lbl.noDataFound = No data found
 frmelmnts.lbl.onDiksha = on {instance} on
 frmelmnts.lbl.opndbtch = Open Batches
 frmelmnts.lbl.orgCode = Org Code
 frmelmnts.lbl.orgId = OrgId
 frmelmnts.lbl.orgType = Org Type
-frmelmnts.lbl.organization = Organization
 frmelmnts.lbl.orgname = Organization Name
 frmelmnts.lbl.orgtypes = Organization Type
-frmelmnts.lbl.originalAuthor = Original Author
-frmelmnts.lbl.ownership = Ownership
 frmelmnts.lbl.participants = Participants
 frmelmnts.lbl.password = Password
 frmelmnts.lbl.pdfcontents = Pdf contents
-frmelmnts.lbl.percentage = Percentage
-frmelmnts.lbl.permanent = Permanent
 frmelmnts.lbl.phone = Mobile number
 frmelmnts.lbl.phoneNumber = Mobile Number
 frmelmnts.lbl.phoneOrEmail = Enter mobile number or email address
 frmelmnts.lbl.phoneVerfied = Mobile number verified
-frmelmnts.lbl.phonenumber = Mobile Number
-frmelmnts.lbl.pincode = Pin Code
-frmelmnts.lbl.playContent = Play Content
 frmelmnts.lbl.plslgn = This session has expired. Login again to continue using${instance}.
 frmelmnts.lbl.preferredLanguage = Preferred Language
 frmelmnts.lbl.previous = Previous
-    
 frmelmnts.lbl.processid = Process ID
 frmelmnts.lbl.profilePopup = To discover relevant content, update the following details:
 frmelmnts.lbl.provider = OrgProvider
-frmelmnts.lbl.publicFooterGetAccess = The {instance} platform offers teachers, students and parents engaging learning material relevant to the prescribed school curriculum. Download the {instance} app and scan QR code in your textbooks for easy access to your lessons.
 frmelmnts.lbl.reEnterPassword = Re-enter the password
 frmelmnts.lbl.publishedBy = Published by
-frmelmnts.lbl.publishedOnInstanceName = Published on {instance} by
```

Comment:
this is used in multiple places. please check. 
frmelmnts.lbl.publishedOnInstanceName